### ISSUE TITLE
micro:bit documentation clarification and possible incompatibility with v1.5 micro:bit

### ISSUE THREAD

**Issue by ghost** [espruino/EspruinoDocs#537]
The documentation for the `acceleration()` and `compass()`commands isn't clear - could you please add a program on how to use them? 

Edit: Maybe the javascript code isn't working with the 1.5 'bit I have with the LSM303AGR?

2nd edit: I'm guessing this as I'm getting this error: `Uncaught InternalError: I2C Write Error 33281`

**Comment by gfwilliams** [comment]
Hi - when working correctly the functions will provide an object containing `{x,y,z}` values which you can then use in your code.

```
setInterval(function() {
  var a = acceleration();
  if (a.z>0) show(1)
  else show(2);
}, 100);
```

However, it looks like support hasn't been added for the updated micro:bit's design yet.

Sadly as there's been surprisingly little interest in our work on it (by users or the micro:bit foundation) getting it fixed is quite low priority at the moment.

**Comment by gfwilliams** [comment]
This is all fixed now - new micro:bit 1.5 is supported
